packages/foreman/rubygem-sassc/rubygem-sassc.spec

```diff
@@ -22,9 +25,16 @@ Requires: %{?scl_prefix}rubygem(ffi) < 2
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}ruby-devel >= 2.0.0
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix}rubygem(ffi) >= 1.9
+BuildRequires: %{?scl_prefix}rubygem(ffi) < 2
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
+BuildRequires: libsass
+Requires: libsass
+# We explicitly point it to libsass.so.0 in %build
+Requires: libsass.so.0()
```

Comment:
@ekohl what is this? :)
It's not in the Fedora spec (https://src.fedoraproject.org/rpms/rubygem-sassc/blob/master/f/rubygem-sassc.spec) which does similar shenanigans to use the system libsass.

Reply:
It was my thinking weird idea to ensure that `libsass.so.0` really does exist. If libsass is bumped to `.so.1` then it would break. Perhaps there's another mechanism to really require this.

Reply:
Maybe the braces are actually incorrect and it should just be `libsass.so.0`

Reply:
I tried both, two sets of braces and no braces, and still no cigar, so I dropped this for now.